src/main.cpp

```diff
@@ -42,43 +42,19 @@
 #include "jsbsim_bridge.h"
 
 int main(int argc, char *argv[]) {
-  if (argc < 5) {
-    cout << "This is a JSBSim integration for PX4 SITL/HITL simulations" << std::endl;
-    cout << "   Usage: " << argv[0] << "<aircraft_path> <aircraft> <config> <scene> <headless>" << endl;
-    cout << "       <aircraft_path>: Aircraft directory path which the <aircraft> definition is located e.g. "
-            "`models/Rascal`"
-         << endl;
-    cout << "       <aircraft>: Aircraft file to use inside the <aircraft_path> e.g. Rascal110-JSBSim" << endl;
-    cout << "       <config>: Simulation config file name under the `configs` directory e.g. rascal" << endl;
-    cout << "       <scene>: Location / scene where the vehicle should be spawned in e.g. LSZH" << endl;
-    cout << "       <headless>: Headless option for flightgear visualiztion 1: enable 0: disable" << endl;
-    return -1;
-  }
+  // Path to config file
+  std::string path = std::string(JSBSIM_ROOT_DIR) + "/configs/" + std::string(argv[3]) + ".xml";
+
+  // Parse Configurations
+  ConfigurationParser *config = new ConfigurationParser();
```

Comment:
You can keep this on the stack
```suggestion
  ConfigurationParser config;
```